sites/bizbash/browser/inquiry-form.vue

```diff
@@ -0,0 +1,220 @@
+<template>
+  <form v-if="incomplete" @submit.prevent="submit">
+    <div class="form-group">
+      <label for="firstName">First Name</label>
+      <input
+        id="firstName"
+        v-model="firstName"
+        type="text"
+        class="form-control"
+        required
+      >
+    </div>
+    <div class="form-group">
+      <label for="lastName">Last Name</label>
+      <input
+        id="lastName"
+        v-model="lastName"
+        type="text"
+        class="form-control"
+        required
+      >
+    </div>
+    <div class="form-group">
+      <label for="email">Email address</label>
+      <input
+        id="email"
+        v-model="email"
+        type="email"
+        class="form-control"
+        required
+      >
+    </div>
+    <div class="form-group">
+      <label for="phone">Phone Number</label>
+      <input
+        id="phone"
+        v-model="phone"
+        type="text"
+        class="form-control"
+        required
+      >
+    </div>
+    <div class="form-group">
+      <label for="company">Company Name</label>
+      <input
+        id="company"
+        v-model="company"
+        type="text"
+        class="form-control"
+      >
+    </div>
+    <div class="form-group">
+      <label for="jobTitle">Job Title</label>
+      <input
+        id="jobTitle"
+        v-model="jobTitle"
+        type="text"
+        class="form-control"
+        required
+      >
+    </div>
+    <div class="form-group">
+      <label for="country">Country</label>
+      <input
+        id="country"
+        v-model="country"
+        type="text"
+        lass="form-control"
+        required
+      >
+    </div>
+    <div class="form-group">
+      <label for="postalCode">ZIP/Postal Code</label>
+      <input
+        id="postalCode"
+        v-model="postalCode"
+        type="text"
+        class="form-control"
+      >
+    </div>
+
+    <div class="form-group">
+      <label for="eventType">Event Type</label>
+      <select
+        id="eventType"
+        v-model="eventType"
+        class="form-control"
+        required
+      >
+        <option disabled value="">
+          Please select...
+        </option>
+        <option v-for="eventType in eventTypes" :key="eventType">
+          {{ eventType }}
+        </option>
+      </select>
+    </div>
+
+    <div class="form-group">
+      <label for="numberOfGuests">Number of Guest(s)</label>
+      <textarea
+        id="numberOfGuests"
+        v-model="numberOfGuests"
+        type="numberOfGuests"
+        class="form-control"
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="eventDate">Event Date</label>
+      <textarea
+        id="eventDate"
+        v-model="eventDate"
+        type="eventDate"
+        class="form-control"
+      />
+    </div>
+
+    <div class="form-group">
+      <label for="eventDescription">Describe Your Event</label>
```

Comment:
Should add an asterisk to required field labels or, better yet, use a required-field class and do a `.required-field::after` content style